packages/core/src/components/fields/SchemaField.js

```diff
@@ -322,6 +322,7 @@ function SchemaFieldRender(props) {
     id,
     label,
     hidden,
+    onChange: props.onChange,
```

Comment:
Small thing: for consistency with the other props, can we pass in a variable called `onChange` that is set equal to `props.onChange`?

Reply:
Sure thing! Updated in b32c659.